plugin/src/main/webapp/icons/LICENSE.txt

```diff
@@ -259,3 +259,10 @@ The Veracode 24x24 & 48x48 icons are licensed under the MIT license.
 https://github.com/jenkinsci/veracode-scan-plugin/tree/master/src/main/webapp/icons
 
 ------------------------------------------------------------------------------
+
+Polyspace icon:
+
+The Polyspace icon is based on tool logo from https://w...mathworks.com/products/polyspace.html
+Licensed by The MathWorks, Inc
```

Comment:
What is the actual license? Are we allowed to add it to our MIT based project?

Reply:
After checking the licence file, we are not sure if we can use it this way. I will remove the icon file and push the changes again, will that be okay ?

Reply:
Yes, this would be more safe. Image licensing is always problematic...